listeners.py

```diff
@@ -41,6 +55,15 @@ def test(value: Any) -> Optional[bool]:
             return test(CopilotPlugin.get_account_status().is_authorized)
         return None
 
+    def on_post_text_command(self, command_name: str, args: Optional[Dict[str, Any]]) -> None:
+        if command_name != "auto_complete":
+            return
+
+        _, session = self._get_session()
+
+        if session and get_setting(session, "hook_to_auto_complete_command"):
+            self.view.run_command("copilot_ask_completions")
```

Comment:
I just realsied, that I can do `plugin.request_get_completions(self.view)` directly here

Reply:
Seems ok. `request_get_completions` has authorization check.